crates/bevy_reflect/derive/src/utility.rs

```diff
@@ -0,0 +1,454 @@
+//! General-purpose utility functions for internal usage within this crate.
+
+use crate::derive_data::ReflectMeta;
+use bevy_macro_utils::{
+    fq_std::{FQAny, FQOption, FQSend, FQSync},
+    BevyManifest,
+};
+use proc_macro2::{Ident, Span, TokenStream};
```

Comment:
This entire file got added in. Why is that?

Reply:
That was me failing to rebase properly.